client/src/pages/dashboard/Dashboard.js

```diff
@@ -131,6 +133,17 @@ function Dashboard() {
     } 
   }
 
+  function postComment() {
+    axios({
+      method: 'post',
+      url: 'http://localhost:8000/post_comment',
+      data: {
+        comment_text: document.getElementById("textbox").value
```

Comment:
In React, it is good convention to use state and "controlled components" to keep track of the value of a textbox rather than accessing the HTML object directly.
Take a look at the "textarea" section of this article for more details: https://reactjs.org/docs/forms.html
(just note what whenever an article uses `this.setState()`, we use hooks instead so your code will have something like `setCommentText()` from a hook rather than `this.setState()`)

Reply:
Also, we want to encourage good coding practices of adding error handling to all HTTP requests. See slack for how you can wrap this command in a try/catch block in case the POST request fails. For now, feel free to just console.log that a message occured